cluster/images/hyperkube/master.json

```diff
@@ -37,6 +60,25 @@
               "--master=127.0.0.1:8080",
               "--v=2"
         ]
+    },
+    {
+      "name": "setup",
```

Comment:
Yep :). However, the certificate generation is failing for me, see
https://github.com/kubernetes/kubernetes/issues/18227

On Friday, December 4, 2015, Mike Danese <notifications@github.com> wrote:

> In cluster/images/hyperkube/master.json
> <https://github.com/kubernetes/kubernetes/pull/17262#discussion_r46744173>
> :
>
> > @@ -37,6 +60,25 @@
> >                "--master=127.0.0.1:8080",
> >                "--v=2"
> >          ]
> > +    },
> > +    {
> > +      "name": "setup",
>
> Works, doesn't it? :) We can make it cleaner with init containers which
> might make it into 1.2.
> #1589 <https://github.com/kubernetes/kubernetes/issues/1589>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17262/files#r46744173>.
>